scripts/operations/node_management/Add_River_Cabinets/update_ncn_etc_hosts.py

```diff
@@ -0,0 +1,250 @@
+#! /usr/bin/env python3
+# MIT License
+#
+# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import argparse
+import datetime
+import json
+import http
+import os
+import requests
+import subprocess
+import sys
+import urllib3
+
+SLS_URL = None
+
+# TODO need to create a common library between this script and the add_management_ncn.py script
+# as a lot of these helper functions were taking from it
+
+#
+# HTTP Action stuff
+#
+def print_action(action):
+    if action['error'] is not None:
+        print(f"Failed:  {action['method'].upper()} {action['url']}. {action['error']}")
+        # print(json.dumps(action["response"], indent=2))
+    elif action['status'] is not None:
+        print(f"Called:  {action['method'].upper()} {action['url']} with params {action['params']}")
+    else:
+        print(f"Planned: {action['method'].upper()} {action['url']}")
+
+    # if action.get('body'):
+    #     print(json.dumps(action.get('body'), indent=2))
```

Comment:
If this is commented, why include it in the code we're checking in?
```suggestion
```

Reply:
Applied here: https://github.com/Cray-HPE/docs-csm/pull/2344/commits/ab3c45572fa5b86e6e61bbd196c6734919b6dae8#diff-4792c55af95856143cc98b5cdc21f3e87bde030c92a6c3ac690606a61f76e7a4L51-L52